I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 193, in script
    python:
  File "renpy/common/00start.rpy", line 194, in <module>
    renpy.execute_default_statement(True)
  File "game/init.rpy", line 5, in execute_default
    default game_over = false
  File "game/init.rpy", line 5, in <module>
    default game_over = false
NameError: name 'false' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 193, in script
    python:
  File "D:\Home\DL\Documents\perso\renpy\renpy-8.3.4-sdk\renpy\ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Home\DL\Documents\perso\renpy\renpy-8.3.4-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00start.rpy", line 194, in <module>
    renpy.execute_default_statement(True)
  File "D:\Home\DL\Documents\perso\renpy\renpy-8.3.4-sdk\renpy\exports\statementexports.py", line 372, in execute_default_statement
    i.execute_default(start)
  File "game/init.rpy", line 5, in execute_default
    default game_over = false
  File "D:\Home\DL\Documents\perso\renpy\renpy-8.3.4-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/init.rpy", line 5, in <module>
    default game_over = false
NameError: name 'false' is not defined

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.4.24120703
fruitsalad 1.0
Sun Jun  8 17:54:20 2025
